src/index.ts

```diff
@@ -27,7 +26,7 @@ const main = async () => {
       addModules: [ "prisma", "auth", "trpc", "tailwind", "naiveui" ],
       runGitInit: true,
       addCi: "github",
-      runInstall: true
+      runInstall: 0
```

Comment:
Can you explain that change ? I believe it is weird to switch a variable whose type is initially boolean to a number.